### DISCUSSION TITLE
We're using Cidre for screen-sharing in production

### DISCUSSION THREAD

**Discussion by morajabi** [yury/cidre#5] [Show and tell]
Hey. Just wanted to thank you for making `cidre`. It's an awesome piece of tech I've enjoyed using while implementing our hardware-accelerated screen-sharing via ScreenCaptureKit. I'm working on decoding incoming h264 stream via `vt`.
(we're building [Noor](https://noor.to) chat app)

**Comment by morajabi** [comment]
https://x.com/morajabi/status/1778190282103922750

**Reply by yury** [reply]
Very cool. Is it ca::Layer::contents?

**Reply by morajabi** [reply]
Yes I used a CALayer as per your recommendation
